xml/depl_crowbar_extra-features.xml

```diff
@@ -105,4 +105,189 @@ skip_unready_nodes:
+   including them in a pending restart list.  Disabling restarts is
+   enabled/disabled by &barcl; and cannot be done on a service level. In other
+   words, enabling this feature on the &o_blockstore; &barcl; will disable
+   automatic restarts for all &o_blockstore;-* services.
+  </para>
+  <para>
+   Two steps are necessary to activate this feature:
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Enable <literal>disallow_restart</literal> in the &crow; configuration file
+    </para>
+   </step>
+   <step>
+    <para>
+     Set the <literal>disable_restart</literal> flag for a specific &barcl;
+     using <literal>crowbar-client</literal> or API
+    </para>
+   </step>
+  </procedure>
+  <para>
+   <emphasis role="bold">Enabling the Feature</emphasis>
+  </para>
+  <procedure>
+   <step>
+    <para>
+     In <filename>/opt/dell/crowbar_framework/config/crowbar.yml</filename>, set
+     the option <literal>disallow_restart</literal> to <literal>true</literal>.
+    </para>
+    <screen>default: &amp;default
+disallow_restart:
+  enabled: false &lt;&lt;&lt; change to true</screen>
+   </step>
+   <step>
+    <para>
+     The <literal>disable_restart</literal> flag can be set with the &crow;
+     client or with the API.
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       Set Flag with &crow; Client
+      </para>
+      <para>
+       The crowbar client options for this feature are accessed with the
+       <command>crowbarctl services</command> command, and only work for
+       &ostack-bare; services. The options are:
+      </para>
+      <variablelist>
+       <varlistentry>
+        <term>disable_restart</term>
+        <listitem>
+         <para>
+          The parameters are the &barcl; name and the flag value
+          (<literal>true</literal> to disable automatic restart and
+          <literal>false</literal> to enable automatic restart). The command is
+          <literal>crowbarctl services disable_restart
+          <replaceable>BARCLAMP</replaceable> &lt;true|false&gt;</literal>. For
+          example, to disable restart of the &o_ident; &barcl;, enter the
+          command <command>crowbarctl services disable_restart keystone
+          true</command>.
+         </para>
+        </listitem>
+       </varlistentry>
+       <varlistentry>
+        <term>restart_flags</term>
+        <listitem>
+         <para>
+          Used to check the <literal>disable_restart</literal> flag for each
+          barclamp
+         </para>
+         <screen>crowbarctl services restart_flags
+{
+  "nova": true,
+  "keystone": true,
+  "database": true
+}</screen>
+        </listitem>
+       </varlistentry>
+       <varlistentry>
+        <term>list_restarts</term>
+        <listitem>
+         <para>
+          Displays the list of pending restarts. The <literal>pending
+          restart</literal> flag indicates that a service tried to
+          restart due to the &chef; run but it did not due to the automatic
```

Comment:
would add here:

`the pacemaker_service attribute indicates if this service is managed by pacemaker (usually in an HA environment) or its an standalone service managed by systemd(usually in on-HA environments)`

Reply:
Added. PDF attached.
[deployment_customizing.pdf](https://github.com/SUSE-Cloud/doc-cloud/files/2581404/deployment_customizing.pdf)